chain/cosmos/chain_node.go

```diff
@@ -665,7 +669,7 @@ func (tn *ChainNode) AddGenesisAccount(ctx context.Context, address string, gene
 		command = append(command, "genesis")
 	}
 
-	command = append(command, "add-genesis-account", address, amount)
+	command = append(command, "add-genesis-account", address, amount, "--chain-id", tn.Chain.Config().ChainID)
```

Comment:
Is `chain-id` necessary here?

Some tests are showing this flag is not available.